src/components/App.js

```diff
@@ -0,0 +1,65 @@
+import "../styles/App.css";
+import React, { useState, useEffect } from "react";
+import LocationDetails from "./LocationDetails";
+import ForecastSummaries from "./ForecastSummaries";
+import ForecastDetails from "./ForecastDetails";
+import getForecast from "../requests/getForecast";
```

Comment:
A minor point but it would be better to group the different import types together (components, requests, css etc)